Sources/TokamakCore/Views/SecureField.swift

```diff
@@ -34,10 +32,17 @@ extension SecureField where Label == Text {
   ) where S: StringProtocol {
     label = Text(title)
     textBinding = text.projectedValue
-    commitAction = onCommit
+    self.onCommit = onCommit
```

Comment:
Is the convention in `init` to specify `self.` or not?

Reply:
Yes, I personally prefer stored properties to have the same names as arguments, which forces you to use `self`. I didn't mind it as much in your PR, but thought I could make it more consistent here with the names of computed proxy properties.

Reply:
I’m fine either way — feel free to change it or I could change it in the future.